packages/jest-jasmine2/src/reporter.ts

```diff
@@ -141,6 +141,7 @@ export default class Jasmine2Reporter implements Reporter {
     const results: AssertionResult = {
       ancestorTitles,
       duration,
+      failureDetails: [],
```

Comment:
can we push into this below as well somehow?

Reply:
Yes, yes we can :D